frontend/src/metabase/entities/containers/EntityType.jsx

```diff
@@ -3,15 +3,14 @@ import React from "react";
 import { connect } from "react-redux";
 import { bindActionCreators } from "redux";
 
-export default entityType => ComposedComponent => {
```

Comment:
I was actually a bit confused at first since this function is also used as `entityType()` while the first argument was also named `entityType`

I'm also not particularly a fan of unnamed export as some IDE like VSCode couldn't auto-import unnamed export as our desired name. Let's say we want other files to import this function as `entityType` we might as well name the function that?

e.g.
```jsx
export default function entityType() {
  return ComposedComponent => {
```

What do you think?